core/object/script_language_extension.h

```diff
@@ -345,7 +345,8 @@ class ScriptLanguageExtension : public ScriptLanguage {
 	EXBIND2RC(int, find_function, const String &, const String &)
 	EXBIND3RC(String, make_function, const String &, const String &, const PackedStringArray &)
 	EXBIND3R(Error, open_in_external_editor, const Ref<Script> &, int, int)
-	EXBIND0R(bool, overrides_external_editor)
+	EXBIND0RC(bool, overrides_external_editor)
+	// EXBIND0RC(ScriptNameCasing, preferred_file_name_casing)
```

Comment:
Why is this a comment?

Reply:
Check last TODO in the PR description. When I last tried implementing this, I just got compile errors.

Reply:
I think the method can't return an enum. Not sure how it should be handled 🤔

Reply:
> I think the method can't return an enum.

Doesn't seem to be the reason, considering other places seem to do it just fine:
https://github.com/godotengine/godot/blob/fbbd3950c82195e4d1b7d681aea0a9bd2fa67f4b/modules/webrtc/webrtc_data_channel_extension.h#L56

I'll try compiling it again, and maybe look deeper into it. But if it doesn't work I'll just end up using `int` and write a code comment.